google-cloud-storage/src/main/java/com/google/cloud/storage/StorageImpl.java

```diff
@@ -696,6 +696,15 @@ public BlobWriteChannel writer(URL signedURL) {
     return new BlobWriteChannel(getOptions(), signedURL);
   }
 
+  @Override
+  public void abort(WriteChannel channel) {
+    if (!(channel instanceof BlobWriteChannel)) {
```

Comment:
why not make the method argument a BlobWriteChannel then?

Reply:
@elharo BlobWriteChannel is a package-private class. If it were public we could invoke channel.abort() directly.